generators/java/reactor-netty/generator/src/main/kotlin/io/github/fomin/oasgen/java/reactor/netty/ReactorNettyClientWriter.kt

```diff
@@ -93,10 +93,29 @@ class ReactorNettyClientWriter(
                     }
                 }.joinToString("")
 
-                val parameterStrDeclarations = javaOperation.parameters.mapIndexed { index, javaParameter ->
-                    val stringWriteExpression =
-                            converterRegistry[javaParameter.schemaParameter.schema()].stringWriteExpression("param$index")
-                    "String param${index}Str = param$index != null ? $stringWriteExpression : null;"
+                val parameterStrDeclarations = javaOperation.parameters.mapIndexedNotNull { index, javaParameter ->
+                            if (javaParameter.schemaParameter.parameterIn != ParameterIn.HEADER) {
```

Comment:
Why headers are excluded here? What for they are handled specially in next block?

Reply:
> Why headers are excluded here? What for they are handled specially in next block?

Anyway, it should be a new block to add parameters to the header's list